src/arena/Missions/Mission.tsx

```diff
@@ -0,0 +1,80 @@
+import { I18n } from "@aws-amplify/core";
+import classNames from "classnames";
+import { Button } from "@mui/material";
+
+import {
+  EnMission,
+  GenMission,
+  Mission as IMission,
+  Missions,
+} from "../interface";
+
+interface MissionProps
+  extends Pick<
+    Missions,
+    | "lengua"
+    | "setActiveIndex"
+    | "setActiveMission"
+    | "setView"
+    | "setShowProofModal"
+    | "missionBtnText"
+  > {
+  mission: GenMission;
+  id: number;
+}
+
+/**
+ * This component is used to display a mission.
+ */
+function Mission({
+  lengua,
+  mission,
+  missionBtnText,
+  setActiveIndex,
+  setActiveMission,
+  setShowProofModal,
+  setView,
+  id,
+}: MissionProps) {
+  let flexCardClass = classNames("context-card", "block", "second-step-arena");
+  return (
+    <li
+      className={flexCardClass}
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "space-between",
+      }}
+    >
+      <div>
```

Comment:
Nit: can this div be removed/replaced with a React.Fragment (without ruining styling)? If so, please replace it with a fragment.